templates/new_user/tokensendnotify.php

```diff
@@ -0,0 +1,2 @@
+<?php
+p($l->t('Activation link sent successfully.'));
```

Comment:
see my comment about generic error messages

for password reset, I believe we had a message like "an activation link was sent to the email address if one was configured" or something like that. Such message should appear even if no email was configured.

In case of error, it might be ok to display a generic message "Error sending activation link. Please contact the administrator". See how the current password reset does it.

Reply:
Adjusted the wordings for the activation link sent progress.